test/e2e/kubectl/logs.go

```diff
@@ -357,4 +358,31 @@ var _ = SIGDescribe("Kubectl logs", func() {
 		})
 	})
 
+	ginkgo.Describe("specific log stream", feature.PodLogsQuerySplitStreams, func() {
```

Comment:
How are you testing this feature?

I think this feature should be off by default. Do you have any test-infra changes to enable this gate for this test?